x-pack/plugins/enterprise_search/public/applications/app_search/components/relevance_tuning/components/precision_slider/precision_slider.tsx

```diff
@@ -0,0 +1,124 @@
+  EuiPanel,
+  EuiRange,
+  EuiSpacer,
+  EuiText,
+  EuiTitle,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import { DOCS_PREFIX } from '../../../../routes';
+import { RelevanceTuningLogic } from '../../relevance_tuning_logic';
+
+import { STEP_DESCRIPTIONS } from './constants';
+
+export const PrecisionSlider: React.FC = () => {
+  const {
+    searchSettings: { precision },
+  } = useValues(RelevanceTuningLogic);
+
+  const { updatePrecision } = useActions(RelevanceTuningLogic);
+
+  const stepDescription = STEP_DESCRIPTIONS[precision];
+
+  return (
+    <>
+      <EuiTitle size="m">
+        <h2>
+          {i18n.translate(
+            'xpack.enterpriseSearch.appSearch.engine.relevanceTuning.precisionSlider.title',
+            {
+              defaultMessage: 'Precision tuning',
+            }
+          )}
+        </h2>
+      </EuiTitle>
+      <EuiSpacer />
+      <EuiPanel hasShadow={false} paddingSize="none">
+        <EuiText color="subdued">
+          {i18n.translate(
+            'xpack.enterpriseSearch.appSearch.engine.relevanceTuning.precisionSlider.description',
+            {
+              defaultMessage: 'Fine tune the precision vs. recall settings on your engine.',
+            }
+          )}{' '}
+          <EuiLink
+            data-test-subj="documentationLink"
+            href={`${DOCS_PREFIX}/precision-tuning.html`}
+            target="_blank"
+          >
+            {i18n.translate(
+              'xpack.enterpriseSearch.appSearch.engine.relevanceTuning.precisionSlider.learnMore.link',
+              {
+                defaultMessage: 'Learn more.',
+              }
+            )}
+          </EuiLink>
+        </EuiText>
+        <EuiSpacer />
+        <EuiFlexGroup justifyContent="spaceBetween">
+          <EuiFlexItem grow={false}>
+            <strong>
+              {i18n.translate(
+                'xpack.enterpriseSearch.appSearch.engine.relevanceTuning.precisionSlider.recall.label',
+                {
+                  defaultMessage: 'Recall',
+                }
+              )}
+            </strong>
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <strong>
+              {i18n.translate(
+                'xpack.enterpriseSearch.appSearch.engine.relevanceTuning.precisionSlider.precision.label',
+                {
+                  defaultMessage: 'Precision',
+                }
+              )}
+            </strong>
+          </EuiFlexItem>
+        </EuiFlexGroup>
+        <EuiSpacer size="s" />
+        <EuiRange
+          data-test-subj="PrecisionRange"
+          value={precision}
+          onChange={(e) => {
+            updatePrecision(parseInt((e.target as HTMLInputElement).value, 10));
+          }}
+          min={1}
+          max={11}
+          step={1}
+          showRange
+          showTicks
+          fullWidth
+        />
+        {stepDescription && (
+          <>
+            <EuiSpacer size="s" />
+            <EuiPanel
```

Comment:
[not a change request, just me grumbling out loud] I tried putting an `aria-live` region for this to see if I could get VO to read out dynamic changes, but it's not working for some reason :( Ah well, no worries, I think it still works decently as-is!